definitions/features/foreman_proxy.rb

```diff
@@ -1,21 +1,58 @@
 class Features::ForemanProxy < ForemanMaintain::Feature
   metadata do
     label :foreman_proxy
+
+    confine do
+      file_exists?('/etc/dhcp/dhcpd.conf') &&
+        ForemanMaintain::Utils::SmartProxySSL.instance.ssl_certs_exists?
+    end
   end
 
-  attr_reader :dhcpd_conf_file, :cert_path
+  attr_reader :dhcpd_conf_file, :cert_path, :base_url,
+              :default_ssl_cert_options
 
   def initialize
     @dhcpd_conf_file = '/etc/dhcp/dhcpd.conf'
```

Comment:
Should this be in config.yaml? OR Can we have a lookup helper for files? Instead of explicit declaration. Would be good to use across various distros.
Just a thought in case `.conf` files is located at some different locations.